Porting D-PARC

JackBeerman opened the pull request:
demo EM to use model initialization, visualization minor appearance tweaks.

JosephBChoi commented:
@JackBeerman Per discussion on last week, the superclass for the model has been implemented. Please have a look at PR #53 and create `DPARC.py` and `test_DPARC()` in `test_model.py`.

JosephBChoi reviewed (changes requested):
see above

JackBeerman commented:
@JosephBChoi addressed all comments except test_DPARC() in test_model.py. waiting for @chengxinlun to review PR 53. We will have to keep in mind the difference of the Unet file and demo. I suggest utilizing the unet in this PR because it supports both conventional and deformable convolutions.

JosephBChoi commented:
@JackBeerman there's an update on main branch. make sure to pull before making any further improvement on this PR.
